backend/http/http.go

```diff
@@ -558,20 +623,37 @@ func (o *Object) stat(ctx context.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to stat")
 	}
-	t, err := http.ParseTime(res.Header.Get("Last-Modified"))
+	var (
+		fSize int64
+		fTime time.Time
+		fType string
+	)
+	if o.fs.opt.GoIndexDriveIndex >= 0 {
+		var fInfo GoIndexFile
+		dec := json.NewDecoder(res.Body)
+		dec.Decode(&fInfo)
+		fSize = parseInt64(fInfo.Size, -1)
```

Comment:
Yeah I can't figure out why I need to manually parse this. GoIndex directly passes the data from the v3 Google Drive API, but those docs say it's supposed to be a long (unless that's mistaken / longs are strings in that context, which is possible, I've seen more weird things from official Google API docs).